docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
 ---
-title: Introduction to Gatsby basics
+title: Familiarize with Gatsby Building Blocks
 typora-copy-images-to: ./
 ---
 
-Hello fellow Gatsby-er! Welcome to _part one_ of our community Gatsby.js
-tutorial.
+In the [previous section](/tutorial/part-zero/), we got our development environment all ready to go by installing different software and scaffolding out our first Gatsby site using the [“hello world” starter](https://github.com/gatsbyjs/gatsby-starter-hello-world). Now we’re going to take a deeper dive into what exactly the [Gatsby](https://github.com/gatsbyjs/gatsby-starter-default)’s [default starter](https://github.com/gatsbyjs/gatsby-starter-default) gives us.
 
-## What's in this tutorial?
+## Gatsby’s default starter
 
-In this tutorial you'll be gently introduced to the Gatsby development
-environment, how to create component pages, and how to build and deploy Gatsby
-sites.
+In the previous section, we created a site based on the “hello world” starter using the following command:
 
-Sit down, buckle up, and let's get going!
+`gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`
 
-## Check your development environment
+When creating a new Gatsby site, you can use the following command structure to create a new site based on any existing Gatsby starter:
 
-Let's check first that you have everything set up to start creating with Gatsby.
-You will need a recent version of Node.js installed.
+`gatsby new [SITE_DIRECTORY] [URL_OF_STARTER_GITHUB_REPO]`
 
-Node.js is a programming tool for running JavaScript on servers and in your
-computer's terminal. Gatsby is built using Node.js.
+> 💡 See a list of the existing [official and community starters](/docs/gatsby-starters/)!
 
-Open a terminal window. See
-[terminal instructions for Mac users](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/) and
-[terminal instructions for Windows users](https://www.quora.com/How-do-I-open-terminal-in-windows). In your terminal window, type `node --version` and hit ENTER, then `npm --version` and hit ENTER (tip: to run a specified command, you must type the command into your terminal and then press ENTER. Then the command will run).
+If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [default starter](https://github.com/gatsbyjs/gatsby-starter-default). Let’s do that now, to create a new site with the default starter, and explore the output.
```

Comment:
"create a new site with the default starter now and explore the output" is a bit shorter and avoids using "let's" (which includes "us")